applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc

```diff
@@ -19,6 +19,7 @@ The *Developer* perspective in the web console provides you the following option
 ** *Database*: Select the required database service and add it to your application.
 ** *Operator Backed*: Select and deploy the required Operator-managed service.
 ** *Helm chart*: Select the required Helm chart to simplify deployment of applications and services.
+** *Devfile*: Select a Devfile from the Devfile registry to declaratively define a development environment.
```

Comment:
Please decide with @rishumehra if we want to keep Devfiles with D capital, here, in the module below as well as in the Release notes.